test/games_manager_test.rb

```diff
@@ -28,14 +28,52 @@ def test_it_has_attributes
   def test_highest_total_score_dummy_file
     path = "./fixture/games_dummy15.csv"
     game_manager = GamesManager.new(path)
-    
+
     assert_equal 5, game_manager.highest_total_score
   end
 
   def test_lowest_total_score_dummy_file
     path = "./fixture/games_dummy15.csv"
     game_manager = GamesManager.new(path)
-    
+
     assert_equal 1, game_manager.lowest_total_score
   end
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  def test_get_season_games
+    path = "./fixture/games_dummy15.csv"
```

Comment:
for season things with the games.csv we need to use the full file because our dummy file only contains info for one season.